dask_geomodeling/geometry/field_operations.py

```diff
@@ -613,7 +616,8 @@ class Mask(BaseSingleSeries):
     Args:
       source (SeriesBlock): Source SeriesBlock that is going to be updated
       cond (SeriesBlock): Conditional (boolean) SeriesBlock that determines
-        whether features in the source SeriesBlock will be updated.
+        whether features in the source SeriesBlock will be updated. Non-
+        boolean data will be converted using data -> True, nodata -> False.
```

Comment:
Same thing about nodata / data. And there is an issue with description right above this modification we can tackle in one go: "correspond to a True value in the conditional are left unchanged. " - that should be False instead of True.